src/content/docs/en/guides/integrations-guide/db.mdx

```diff
@@ -92,6 +92,31 @@ export default defineDb({
 })
 ```
 
+## Configuration
+
+### `mode`
+
+Astro DB supports different driver modes for production deployments, by default it uses a node-based libSQL driver. When deploying to a serverless environment's that don't support all features of Node.js, such as Cloudflare Workers, you can switch to the web libSQL driver allowing for Astro DB to continue functioning in these environments.
+
+To configure the driver mode, set the `mode` property in your `astro.config.mjs` file:
+
+```ts title="astro.config.mjs" ins={7}
+import { defineConfig } from 'astro/config';
+import db from '@astrojs/db';
+
+export default defineConfig({
+  integrations: [
+   db({
+     mode: 'web'
+   })
+  ]
+});
+```
+
+:::note
```

Comment:
See my comment above, I'd remove this note in favour of more fully explaining the two modes themselves.